### PR TITLE
Bug#1459010  fixes

### PR DESCRIPTION
**Pull request by gaurav-nelson**
Added a new sections for prerequisites.

### CODE REVIEW

**Comment by gaurav-nelson** [comment]
@tanaka-takayoshi Can you please review?

**Comment by gaurav-nelson** [comment]
@tanaka-takayoshi bump

**Comment by tanaka-takayoshi** [comment]
@gaurav-nelson Sorry I'm late. It looks good to me.

**Comment by gaurav-nelson** [comment]
@bfallonf PTAL

File: install_config/persistent_storage/persistent_storage_azure_file.adoc [hunk 1]
```diff
@@ -26,6 +26,23 @@ High availability of storage in the infrastructure is left to the underlying
 storage provider.
 ====
 
+[[azure-file-before-you-begin]]
+== Before you begin
+. Install `samba-client`, `samba-common` and `cifs-utils` on all nodes:
```
Comment (bfallonf) [inline install_config/persistent_storage/persistent_storage_azure_file.adoc]: comma after `samba-common`

**Comment by bfallonf** [comment]
@gaurav-nelson one comment from me. Good otherwise! :goat:

**Comment by gaurav-nelson** [comment]
No revision history needed.

**Pull request merged**